docs/vNext/tutorials_fabric_existing.md

```diff
@@ -0,0 +1,724 @@
+
+This is what should be added: 
+
+```bash
+"adminPrivateKey": {
+    "path": "../fabric-samples/test-network/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/keystore/priv_sk"
+    },
+"signedCert": {
+    "path": "../fabric-samples/test-network/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/signcerts/Admin@org1.example.com-cert.pem"
+    }
+```
+
+And this is what the `organizations` object should look like now:
+
+```bash
+    "organizations":{
+        "Org1": {
+            "mspid": "Org1MSP",
+            "peers": [
+                "peer0.org1.example.com"
+            ],
+            "certificateAuthorities": [
+                "ca.org1.example.com"
+            ],
+            "adminPrivateKey": {
+                "path": "../fabric-samples/test-network/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/keystore/priv_sk"
+            },
+            "signedCert": {
+                "path": "../fabric-samples/test-network/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/signcerts/Admin@org1.example.com-cert.pem"
+            }
+        }
+    },
+```
+
+### Resulting file:
+
+The network configuration file should now be fully populated. It can be useful to take time to look over and ensure that the paths to the certificate and keys are correct. 
+
+The whole file should resemble like this: 
+
+```bash
+{
+    "version": "1.0",
+    "caliper": {
+        "blockchain": "fabric"
+    },
+    "clients": {
+        "Admin@org1.example.com": {
+            "client": {
+                "credentialStore": {
+                    "path": "tmp/hfc-kvs/org1",
+                    "cryptoStore": {
+                        "path": "tmp/hfc-kvs/org1"
+                    }
+                },
+                "organization": "Org1",
+                "clientPrivateKey": {
+                    "path": "../fabric-samples/test-network/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/keystore/priv_sk"
+                },
+                "clientSignedCert": {
+                    "path": "../fabric-samples/test-network/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp/signcerts/Admin@org1.example.com-cert.pem"
+                },
+                "connection": {
+                    "timeout": {
+                        "peer": {
+                            "endorser": "300"
+                        }
+                    }
+                }
+
+            }
+        }
+    },
+    "channels": {
+        "mychannel": {
+            "created" : true,
+            "contracts": [
+                {
+                    "id":"fabcar",
```

Comment:
test-network might not be deploying fabcar as fabcar anymore, I just tried it and it was called `basic`